components/kyma-environment-broker/cmd/broker/upgrade_cluster_test.go

```diff
@@ -1,8 +1,16 @@
 package main
 
-//TODO: intentionally disabled, depends on https://github.com/kyma-project/control-plane/pull/1563
```

Comment:
this one was commented out but can be enabled after https://github.com/kyma-project/control-plane/pull/1773 which implemented and cherry-picked relevant parts from https://github.com/kyma-project/control-plane/pull/1563